launchable/commands/subset.py

```diff
@@ -308,9 +308,16 @@ def get_payload(
 
         def run(self):
             """called after tests are scanned to compute the optimized order"""
+            if not is_get_tests_from_previous_sessions and len(self.test_paths) == 0:
+                click.echo(
+                    click.style(
+                        "ERROR: subset candidates are empty. Please set subset candidates or use `--get-tests-from-previous-sessions` option",  # noqa E501
+                        fg="red"),
+                    err=True)
+                exit(1)
 
             # When Error occurs, return the test name as it is passed.
-            original_subset = self.test_paths
+            original_subset = []
```

Comment:
I think we can revert this as well? Falling back to full run is orthogonal to this PR. (Also aligned with the comment above).